inst/tests/tests.Rraw

```diff
@@ -6031,6 +6031,31 @@ thisDT <- copy(DT)[, c("aaa", "b") := 2]
 test(1419.58, indices(thisDT), c("a", "ab"))
 test(1419.59, allIndicesValid(thisDT), TRUE)
 
+## setkey on same col as index before
+DT <- data.table(a =c(4,1,3,9,2,1,8,7,6,5),
+                 aaa = c(1,1,2,2,2,1,1,2,2,2))
+setindex(DT, a)
+test(1419.60, allIndicesValid(DT), TRUE)
+setindex(DT, NULL)
+setkey(DT, a)
+test(1419.61, DT$a, c(1,1,2,3,4,5,6,7,8,9))
+setkey(DT, NULL)
+setindex(DT, a)
+test(1419.62, setkey(DT, a, verbose=TRUE), output="using existing index for a")
+
+# check setkey incase of existing multiple indexes
+DT <- data.table(a = c(3,3,4,4,5,6,1,1,7,2,2),
```

Comment:
`a` has 11 items here but `aaa` and `bbb` have 10 so there's a warning when running test.data.table() runs in dev and when users would run test.data.table().  It's a lot easier and faster to run locally first before pushing. See the `cc.R` script in the root of the project : that's what I use in dev.